src/App.tsx

```diff
@@ -1,61 +1,141 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId) || null,
+}));
+
+export const App: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [todoTitle, setTitle] = useState('');
+  const [visibleTodos, setTodos] = useState(todos);
+  const [noUsers, setNoUsers] = useState(false);
+  const [noTodos, setNoTodos] = useState(false);
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setNoTodos(false);
+    setTitle(value);
+  };
+
+  const handleChangeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    setNoUsers(false);
+    setSelectedUser(getUserById(Number(value)));
+  };
+
+  const isCompleted = () => {
+    return false;
+  };
+
+  const handleAddTodo = (event: { preventDefault: () => void; }) => {
```

Comment:
just get type to a whole event
```suggestion
  const handleAddTodo = (event: ... ) => {
```